addons/mail/models/discuss/discuss_channel_rtc_session.py

```diff
@@ -59,10 +59,13 @@ def create(self, vals_list):
                     "start_call_message_id": message.id,
                 },
             )
-            Store(bus_channel=channel).add(message, [Store.Many("call_history_ids", [])]).bus_send()
+            stores[channel].add(message, lambda res: res.many("call_history_ids", []))
+        for store in stores.values():
+            store.bus_send()
         return rtc_sessions
 
     def unlink(self):
+        stores = lazymapping(lambda channel: Store(bus_channel=channel))
```

Comment:
Note: this optimizes (and explains the change in rtc tests) to send a single store/bus for the same target (in this flow) for both rtc updates and history update.